Volontulo/app/src/main/java/com/stxnext/volontulo/model/Offer.java

```diff
@@ -19,6 +19,9 @@
         value = Parcel.Serialization.BEAN,
         analyze = { Offer.class })
 public class Offer extends RealmObject {
+
+    public static String IMAGE_RESOURCE = "IMAGE-RESOURCE";
```

Comment:
This field I will be mark with `@Ignore` annotation to prevent store this field in database